libkbfs/bserver_disk.go

```diff
@@ -7,11 +7,12 @@ package libkbfs
 import (
 	"errors"
 	"fmt"
-	"io/ioutil"
 	"os"
 	"path/filepath"
 	"sync"
 
+	"github.com/keybase/kbfs/ioutil"
```

Comment:
Hmm, I guess goimports doesn't automatically reformat. Went through and fixed them all up.